collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -453,10 +462,10 @@
 QUERY_STANDBY = {
     DEFAULT: """
 SELECT
-    application_name
-FROM pg_stat_replication
-WHERE application_name IS NOT NULL
-GROUP BY application_name;
+    coalesce(prs.slot_name, psr.application_name) application_name
```

Comment:
```suggestion
    COALESCE(prs.slot_name, psr.application_name) application_name
```

Let's make the function uppercase